src/components/override-sdk/template/DefaultForm/DefaultForm.tsx

```diff
@@ -94,10 +94,17 @@ export default function DefaultForm(props) {
 
   useEffect(()=>{
     if(containerName === 'Declaration'){
+      // Get current context
       const context = PCore.getContainerUtils().getActiveContainerItemName(`${PCore.getConstants().APP.APP}/primary`);
-      const declarationText1 = PCore.getStoreValue('.DeclarationText1', 'caseInfo.content.Claim', context);
-      const declarationWarning1 = PCore.getStoreValue('.DeclarationWarning1', 'caseInfo.content.Claim', context);
-      setDeclaration({text1 : declarationText1, warning1: declarationWarning1});
+      // Get is documentation required flag value
+      const isDocumentationRequired = PCore.getStoreValue('.IsDocumentationRequired', 'context_data.caseInfo.content.Claim.summary_of_when_conditions__', context);
```

Comment:
is `context_data` is required as this is not used in below operations?

Reply:
I'm not sure what you mean as all I'm getting here is a value for a key? It's the only place I can find that flag. Please see the attached tree. 

![image](https://github.com/norm-l/odx/assets/149371097/663e6f8d-2664-4b9c-aed2-90559cfde518)

Reply:
I understand. For some reason the flag only exists in a different location to the warning and text. I don't know why. I'll resolve this for now.